src/Microsoft.Diagnostics.ExtensionCommands/Output/ColumnKind.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.Diagnostics.ExtensionCommands.Output
+{
+    internal static class ColumnKind
+    {
+        private static Column? s_text;
+        private static Column? s_hexOffset;
+        private static Column? s_hexValue;
+        private static Column? s_dumpObj;
+        private static Column? s_integer;
+        private static Column? s_dumpHeapMT;
+        private static Column? s_listNearObj;
+        private static Column? s_dumpDomain;
+        private static Column? s_thread;
+        private static Column? s_integerWithoutComma;
+        private static Column? s_humanReadable;
+        private static Column? s_range;
+
+        private static int PointerLength => IntPtr.Size * 2;
+
+        /// <summary>
+        /// A pointer, displayed as hex.
+        /// </summary>
+        public static Column Pointer { get; } = new(Align.Right, PointerLength, Formats.Pointer);
```

Comment:
If you making the process size pointer assumption, why can't you cache this in a static like the rest of Columns?

Reply:
This one is left over from a previous refactoring.  I'll fix that depending on what I do about IntPtr.Size.